scripts/all-locales.js

```diff
@@ -47,13 +52,13 @@ var LANGS = [
 var RTL_LANGS = ['ar', 'fa', 'he', 'ur'];
 
 
+// eventually might have to do a cp -r type thing
 var nothingBurger = function (lang) {
   var dir = path.join('repair', lang);
-  var file = path.join(dir,"index.html");
+  var outfile = path.join(dir,"index.html");
+  var basefile = path.join('scripts', 'locale_base', 'index.html')
```

Comment:
I would probably put the source declaration above the destination declaration, and perhaps inline the dir variable.